unite_contranges.py

```diff
@@ -0,0 +1,146 @@
+#Allison Towner
+#09 October 2020
+
+#continuum_selections.py in analysis code is located at:
+#in ALMA_IMF/reduction/analysis/
+
+def merge_contdotdat(field='',band=''):
+
+	import numpy as np
+	import sys
+	sys.path.append('.')
+	from parse_contdotdat import parse_contdotdat
+
+	#Read in metadata from .json file
+	basepath = '/orange/adamginsburg/ALMA_IMF/2017.1.01355.L'
```

Comment:
this should be an input to the function, so that if we have to change the path, we don't have to change something hard-coded
(i.e., `def f(..., basepath='...')`